test/capitalize.spec.js

```diff
@@ -1,13 +1,13 @@
 'use strict';
 
-var capitalizeFirst = require('../').capitalizeFirst;
+var capitalize = require('../').capitalize;
 var tape = require('tape');
 var Handlebars = require('handlebars');
 
-Handlebars.registerHelper(capitalizeFirst.name, capitalizeFirst);
+Handlebars.registerHelper(capitalize.name, capitalize);
```

Comment:
@mrgerardorodriguez No idea, that's just how every helper is structured and I copied it. 😆